### PR TITLE
WRQ-31515: Fixed `Scroller`, `Slider`, `VirtualList` and `VirtualGridList` to set prop value to default when `undefined` is passed for the prop value

### PR DESCRIPTION
**Pull request by juwonjeong**
### Checklist

* [x] I have read and understand the [contribution guide](http://enactjs.com/docs/developer-guide/contributing/)
* [x] A [CHANGELOG entry](http://enactjs.com/docs/developer-guide/contributing/changelogs/) is included
* [ ] At least one test case is included for this feature or bug fix
* [x] Documentation was added or is not needed
* [ ] This is an API breaking change

### Issue Resolved / Feature Added
[//]: # (Describe the issue resolved or feature added by this pull request)
In https://github.com/enactjs/sandstone/pull/1613, we removed defaultProps and implemeted this feature manually.
After this implementation, if `undefined` is passed for a prop value, the prop value remains undefined.
However, defaultProps(https://legacy.reactjs.org/docs/typechecking-with-proptypes.html#default-prop-values) sets prop value to default value when undefined value is passed for the prop.

### Resolution
[//]: # (Does the code work as intended?)
[//]: # (What is the impact of this change and *why* was it made?)
Fix `Scroller`, `Slider`, `VirtualList` and `VirtualGridList` to set prop value to default value when `undefined` is passed for the prop value

### Additional Considerations
[//]: # (How should the change be tested?)
[//]: # (Are there any outstanding questions?)
[//]: # (Were any side-effects caused by the change?)
- I checked React's defaultProps for falsy values(null, undefined, false, NaN, 0, -0, "") and found that it sets the prop value to its default value only when undefined is passed.
https://react.dev/reference/react/Component#static-defaultprops

- I found that the default value of wheelInterval in controls of Slider sampler was not loading properly. So I fixed it in this PR.
https://github.com/enactjs/sandstone/pull/1658/files#diff-4f80023ae6bb7b165147824d15263e1a011ab278fdf4a5f6ddbea07fecca1a31R471

### Links
[//]: # (Related issues, references)
WRQ-31515

### CODE REVIEW

**Comment by MikyungKim** [comment]
@juwonjeong, Please resolve the merge conflict.

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
 
 The following is a curated list of changes in the Enact sandstone module, newest changes on the top.
 
+## [unreleased]
+
+### Fixed
+
+- `sandstone/Scroller`, `sandstone/Slider`, `sandstone/VirtualList` and `sandstone/VirtualList.VirtualGridList` to set prop value to default when `undefined` is passed for the prop value
```
Comment (MikyungKim) [inline CHANGELOG.md:9]:
```suggestion
- `sandstone/Scroller`, `sandstone/Slider`, `sandstone/VirtualList` to have default prop when `undefined` prop is passed
```

File: Slider/Slider.js [hunk 2]
```diff
@@ -467,6 +468,8 @@ const SliderDecorator = compose(
 
 const Slider = SliderDecorator(SliderBase);
 
+Slider.defaultPropValues = sliderDefaultProps;
```
Comment (MikyungKim) [inline Slider/Slider.js:471]: 👍

File: CHANGELOG.md (hunk 1 above)
Comment (MikyungKim) [reply CHANGELOG.md:9]: We can refer `VirtualList` and VirtualGridList` to `sandstone/VirtualList`. :)

Comment (juwonjeong) [reply CHANGELOG.md]: Thank you. I fixed to use sandstone/VirtualList

**Review by MikyungKim — APPROVED** [review]
LGTM

**Review by 0x64 — APPROVED** [review]

**Pull request merged**